.travis.yml

```diff
@@ -16,15 +16,7 @@ env:
     - PYTHON=""
   matrix:
     - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
-    - CONDA_JL_VERSION="2"
 jobs:
-  allow_failures:
-    - julia: nightly
```

Comment:
We should still allow nightly failures

Reply:
Oh whoops. I didn't notice the os part was a connected to the env. I thought they were all combined together.